hack/install.sh

```diff
@@ -79,7 +79,7 @@ if [ -z "$lsb_dist" ] && [ -r /etc/os-release ]; then
 	lsb_dist="$(. /etc/os-release && echo "$ID")"
 fi
 
-lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"
+lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]'|  cut -d' ' -f1)"
```

Comment:
the distort is usually echoed as "elementary OS", the cut is to grep
elementary only.

On Mon, Apr 6, 2015 at 9:47 PM Tianon Gravi <notifications@github.com>
wrote:

> In hack/install.sh
> <https://github.com/docker/docker/pull/11857#discussion_r27807409>:
>
> > @@ -79,7 +79,7 @@ if [ -z "$lsb_dist" ] && [ -r /etc/os-release ]; then
> >  	lsb_dist="$(. /etc/os-release && echo "$ID")"
> >  fi
> >
> > -lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]')"
> > +lsb_dist="$(echo "$lsb_dist" | tr '[:upper:]' '[:lower:]'|  cut -d' ' -f1)"
>
> Why the extra cut here?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/11857/files#r27807409>.
>